pandas/core/ops.py

```diff
@@ -582,6 +751,35 @@ def _get_op_name(op, special):
 DataFrame.{reverse}
 """
 
+_flex_comp_doc_FRAME = """
+Flexible wrappers to comparison operators (specifically ``{name}``).
+
+Wrappers (``eq``, ``ne``, ``le``, ``lt``, ``ge``, ``gt``) are equivalent to
+operators (``==``, ``=!``, ``<=``, ``<``, ``>=``, ``>``) with support to choose
+axis (rows or columns) for comparison.
+
+Parameters
+----------
+other : DataFrame
+axis : {{0, 1, 'columns', 'rows'}}
+level : int or name
+    Broadcast across a level, matching Index values on the
+    passed MultiIndex level
+
+Returns
+-------
+result : DataFrame
+    Consisting of boolean values
+
+Examples
+--------
+{df_examples}
+
+See also
+--------
+{reverse}
```

Comment:
`See Also` should go before examples. Also, note the capital `A`.

Can you run `./scripts/validate_docstring.py pandas.DataFrame.eq` after the next set of changes to validate these details?

Reply:
We still need to add several validations to that script. The order of the sections is one of them.